### PR TITLE
Fix #8661 - Add Custom Connector docs

### LINKED ISSUE #8661: Custom Connector Docs
- Docs should be easily searchable
- Explain what are the basic requirements
- How to package the code/image
- How to use it from the UI

### PR DESCRIPTION
**Pull request by pmbrull**
### Describe your changes :

Fix #8661

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-docs/content/connectors/custom-connectors.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+title: Custom Connectors
+slug: /connectors/custom-connectors
+---
+
+# Custom Connectors
+
+Each of the services support providing a Custom Connector. It should be a Python class available from the Ingestion
+image. It should also match specific constraints on the methods to implement and how to send the Entities to be
```
Comment (TeddyCr) [inline openmetadata-docs/content/connectors/custom-connectors.md:9]: should we generalize that to something like "where your ingestion workflows are running (e.g. Docker Image, EC2 instance, etc.)"

File: openmetadata-docs/content/connectors/custom-connectors.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+
+## Step 1 - Prepare your Connector
+
+A connector is a class that extends from `metadata.ingestion.api.source.Source`. It should implement
+all the required methods ([docs](https://docs.open-metadata.org/sdk/python/build-connector/source#for-consumers-of-openmetadata-ingestion-to-define-custom-connectors-in-their-own-package-with-same-namespace)).
+
+In [connector/my_awesome_connector.py](https://github.com/open-metadata/openmetadata-demo/blob/main/custom-connector/connector/my_awesome_connector.py) you have a minimal example of it.
+
+Note how te important method is the `next_record`. This is the generator function that will be iterated over
```
Comment (TeddyCr) [inline openmetadata-docs/content/connectors/custom-connectors.md:31]: I think there is a small typo here.

File: openmetadata-docs/content/connectors/custom-connectors.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+other types of Entities is to directly refer to the `ometa` [integration tests](https://github.com/open-metadata/OpenMetadata/tree/main/ingestion/tests/integration/ometa).
+
+## Step 3 - Prepare the package installation
+
+We'll need to package the code so that it can be shipped to the ingestion container and used there. In this demo
+you can find a simple `setup.py` that builds the `connector` module.
+
+## Step 4 - Prepare the Ingestion Image
+
+If you want to use the connector from the UI, the `openmetadata-ingestion` image should be aware of your new package.
```
Comment (TeddyCr) [inline openmetadata-docs/content/connectors/custom-connectors.md:49]: Same as the first comment.

**Review by TeddyCr — COMMENTED** [review]
Left a few comments

**Comment by pmbrull** [comment]
Thanks for checking @TeddyCr, addressed the comments 🙏

**Review by TeddyCr — APPROVED** [review]
LGMT

**Pull request merged**